packages/mgt-components/src/utils/Utils.ts

```diff
@@ -333,3 +333,13 @@ export const getResponseInvalidationTime = (currentInvalidationPeriod: number) =
 export const getIsResponseCacheEnabled = () => {
   return CacheService.config.response.isEnabled && CacheService.config.isEnabled;
 };
+
+/**
+ * Format the availability string
+ *
+ * @returns boolean
+ */
+export const formatAvailability = (availability: string) => {
```

Comment:
is this function used anymore? if not let's delete it

Reply:
Thanks!